sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -196,7 +195,12 @@ void mask_rom_main(void) {
   base_printf("OpenTitan: \"version-tag\"\r\n");
   base_printf("lc_state: %s\r\n", lifecycle_state_name[lc_state]);
 
-  rom_error_t error = mask_rom_try_boot();
+  rom_error_t error = primitive_bootstrap();
```

Comment:
nit: `TODO` with issue number for replacing this with the actual bootstrap?

Reply:
Done.